app/routes/auth.js

```diff
@@ -115,6 +117,16 @@ router.post('/signup', (req, res, next) => {
     });
   }
 
+  if (!firstName || !lastName) {
+    return res.status(400).json({
+      success: false,
+      error: 'no-name',
+      message: 'Please provide both your first and last name.',
+    });
+  }
+
+  const recaptchaResponse = req.body['g-recaptcha-response'] || '';
```

Comment:
Given the condition on line 130, the conditional assignment here is unnecessary.